frontend/src/libs/components/header/styles.module.scss

```
@@ -0,0 +1,21 @@
+.header {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px 61px 31px 62px;
```

Let's use symmetric dimensions for padding 
```suggestion
  padding: 30px 60px;
```